charts/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM debian:bullseye-20230522 as downloader
```

Comment:
What is this Dockerfile used for?

Reply:
I removed it again, it was just a commit errror